qtp_biom/summary.py

```diff
@@ -13,18 +13,21 @@
 
 import qiime2
 from qiime2.plugins.feature_table.visualizers import summarize
+from skbio.tree import TreeNode
+from biom import load_table
 
 
 Q2_INDEX = """<!DOCTYPE html>
 <html>
   <body>
+    %s <!-- summarizing phylogenetic tree, if existent -->
     <iframe src="./support_files/%s" width="100%%" height="850" frameborder=0>
     </iframe>
   </body>
 </html>"""
 
 
-def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis):
+def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis, tree):
```

Comment:
Perhaps worth replacing tree for tree=None so by default it's always None. This should help needing to replace all calls of this method that we don't need to change. However, I _think_ the only call to this method is within this repo and not in many places.

Reply:
excellent idea